plugins/woocommerce-admin/client/customize-store/intro/actions.ts

```diff
@@ -12,16 +12,25 @@ import { customizeStoreStateMachineContext } from '../types';
 import { ThemeCard } from './types';
 import { events } from './';
 
-export const assignThemeCards = assign<
+export const assignThemeData = assign<
 	customizeStoreStateMachineContext,
 	customizeStoreStateMachineEvents // this is actually the wrong type for the event but I still don't know how to type this properly
 >( {
 	intro: ( context, event ) => {
 		const themeCards = (
-			event as DoneInvokeEvent< { themeCards: ThemeCard[] } >
-		 ).data.themeCards;
+			event as DoneInvokeEvent< { themeData: { themes: ThemeCard[] } } >
+		 ).data.themeData.themes;
+
 		// type coercion workaround for now
-		return { ...context.intro, themeCards };
+		return {
+			...context.intro,
+			themeData: {
+				themes: themeCards,
+				_links: {
+					browse_all: '/wp-admin/themes.php',
```

Comment:
Is there any reason not to use `themeData._links.browse_all.href` directly?

Reply:
No reason :) That was a mistake actually. Thank you for catching 👍 

Fixed in [Use browse_all.href and set default value to /:admin-dir/themes.php](https://github.com/woocommerce/woocommerce/pull/40552/commits/a191ca3a2cd9c62e047aa8782287275226440812)